.jenkins/caffe2/bench.sh

```diff
@@ -13,7 +13,8 @@ else
     num_gpus=0
 fi
 
-cmd="$PYTHON $CAFFE2_PYPATH/python/examples/resnet50_trainer.py --train_data null --batch_size 64 --epoch_size 6400 --num_epochs 2"
+caffe2_pypath="$(dirname $(python -c 'import caffe2; print(caffe2.__file__)'))"
```

Comment:
`__file__` is a list, do `__file__[0]`

Reply:
On the Linux docker I tested this on it is a string. I will investigate more

Reply:
It looks like the behavior of __file__ was changed in 3.4 https://bugs.python.org/issue18416 . Maybe this is not a very safe method

Reply:
Sorry I thought you were using `__path__`. `__file__` indeed is a string.